source/experimental/ConsoleRenderer/ConsoleRenderer/ConsoleRender.cpp

```diff
@@ -332,23 +334,23 @@ void RenderBorder(std::vector<std::string> &output, size_t columnWidth)
     bottom.append(ESC "(B");
 
     output[0] = top;
-    //output.insert(output.begin(), top);
+    // output.insert(output.begin(), top);
     output.push_back(bottom);
 }
 
-void RenderToConsole(std::shared_ptr<AdaptiveCards::AdaptiveCard> card, size_t columnWidth, std::vector<std::string> &output)
+void RenderToConsole(std::shared_ptr<AdaptiveCards::AdaptiveCard> card, size_t columnWidth, std::vector<std::string>& output)
 {
     std::vector<std::string> outputVector;
     auto body = card->GetBody();
-    RenderElements(body, columnWidth-4, 1, 1, output); // save 4 width for border
+    RenderElements(body, columnWidth - 4, 1, 1, output); // save 4 width for border
     RenderBorder(output, columnWidth);
-    //printf("\n\n\n");
+    // printf("\n\n\n");
```

Comment:
>// printf("\n\n\n"); [](http://example.com/codeflow?start=4&length=20)

lots of comments that we could erase, as this is a formatting PR maybe we could do it in a different PR?

Reply:
this is on the console renderer so maybe this is not important

Reply:
Yes. this PR only do formatting files with tools, and will have no manual code change on formatted files.